source/content/Character/Classes/Cleric/Miracles/Level 1/Cure Light Wounds.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Cure Light Wounds
+tags:
+  - level_1
+  - cleric
+---
+**Cleric Level 1**  
+**Duration**: Instantaneous  
+**Range**: Touch  
+
+By the power of the Cleric’s faith, this spell restores **1d6 Hit Points** to one damaged character, plus a number of points equal to the level of the caster. For example, if the caster is **5th level**, the spell restores **1d6+5 Hit Points**.  
+
+The spell can also be used to remove **one temporary negative condition** instead of restoring lost Hit Points. For example, if a character is **paralyzed, stunned, or blinded** for a set period of time, this spell will remove any of these conditions. It will **not** work on permanent conditions, and only **one condition (or damage) can be treated per casting** of the spell.  
```

Comment:
does it treat crit wounds which last like for weeks?

Reply:
"It will not work on permanent or long-term conditions (lasting longer than a day), ..."
Is it okay?

Reply:
I find that "cure light wounds" should work for things like bruised ribs n stuff. So i would think about it changing the time required. Like the doc heals more wounds, the cleric might speed up the restoration from such things. For example, you got 1d4 weeks, cleric makes it -1 day for each cast (daily, they should not stack i think)